docs/source/getting-started/index.rst

```diff
@@ -6,11 +6,66 @@ quickstart tutorials here are the perfect starting point. They will guide you
 step-by-step through fundamental concepts, focusing purely on a task that will maximize your understanding of 
 how to use MLflow to solve a particular task. 
 
+`5-minute Quickstart - MLflow Tracking <intro-quickstart/index.html>`_
+----------------------------------------------------------------------
+
+In this brief introductory quickstart on MLflow Tracking, you will learn how to log, register, and load a model for inference. In the process of 
```

Comment:
```suggestion
In this brief introductory quickstart on MLflow Tracking, you will learn how to log training stats (loss, accuracy, etc), then register your model on MLflow server, and finally load the registered model for inference. In the process of 
```

Reply:
Changed to a bullet list to simplify the long sentence